onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -529,14 +531,15 @@ public void setOnFocusChangeListener(OnFocusChangedListener onFocusChangedListen
     final Handler mStopChangedHandler = new Handler();
 
     public void onFocusChanged(final ObaStop stop, final HashMap<String, ObaRoute> routes,
-            final Location location) {
+                               final Location location) {
         // Run in a separate thread, to avoid blocking UI for long running events
         mStopChangedHandler.post(new Runnable() {
             public void run() {
                 if (stop != null) {
                     mFocusStopId = stop.getId();
                     //Log.d(TAG, "Focused changed to " + stop.getName());
                 } else {
+                    mFocusStopId = null;
```

Comment:
@cagryInside Friendly ping on this - could you please take a look in the next few days so we can get this merged?  See above for my notes.